api/desecapi/metrics.py

```diff
@@ -31,7 +31,11 @@ def set(name, *args, **kwargs):
 
 set('desecapi_database_unavailability_count', 'number of times database was unavailable')
 
-#.py metrics
+#pdns.py metrics
+
+set('desecapi_pdns_request_successfull', 'number of times pdns request was successfull')
```

Comment:
1.) successfu**l**, 2.) in the encryption/decryption metrics, you called the metric `..._success`. Let's use the shorter version consistently